docs/rst/tutorials/cloud/router_participant_security_via_xml_profiles/router_participant_security_via_xml_profiles.rst

```diff
@@ -0,0 +1,149 @@
+.. _tutorials_ddsrouter_participant_via_xml_profiles:
+
+ROS 2 Router configuration of DomainParticipants Security via XML profiles
+==========================================================================
+
+.. contents::
+    :depth: 2
+    :local:
+    :backlinks: none
+
+
+Background
+----------
+
+*eProsima ROS 2 Router*, a.k.a `DDS Router <https://github.com/eProsima/DDS-Router>`_, is a cross-platform non-graphical application developed by eProsima that allows users to connect isolated DDS networks.
+These networks could be isolated one to each other due to the Transport Protocol (UDP, TCP, etc.), the Discovery Protocol (Simple, Discovery Server, etc.) or the DDS Domain Id used by each DDS entity.
+Configuring the DDS Router to have one Participant for each isolated network will allow to establish the communication and data transmission between the entities.
+In compliance with the DDS Security specification, Fast DDS provides secure communication by implementing pluggable security at three levels:
+DomainParticipants authentication (`DDS:Auth:PKI-DH plugin <https://fast-dds.docs.eprosima.com/en/latest/fastdds/security/auth_plugin/auth_plugin.html>`_), access control of Entities (`DDS:Access:Permissions plugin <https://fast-dds.docs.eprosima.com/en/latest/fastdds/security/access_control_plugin/access_control_plugin.html>`_), and data encryption (`DDS:Crypto:AES-GCM-GMAC <https://fast-dds.docs.eprosima.com/en/latest/fastdds/security/crypto_plugin/crypto_plugin.html>`_).
+Fast DDS allows security plugins to be activated through the `DomainParticipantQos <https://fast-dds.docs.eprosima.com/en/latest/fastdds/dds_layer/domain/domainParticipant/domainParticipant.html#dds-layer-domainparticipantqos>`_ properties.
+
+In this tutorial, the configuration of two DDS Router Participants will be undertaken through the utilization of XML Profiles, thereby imbuing them with enhanced security features.
+Specifically, these Participants will facilitate communication between two distinct domains, a capability that would be unattainable in the absence of the Router.
```

Comment:
The Router participants allow the communication, so I would change it this way: "Specifically, these Participants allow the communication between two distinct domains, a capability that would be unattainable in the absence of the Router."